### PR TITLE
Create 2020-03-23-coreweekly-week-12-2020.md

### PR DESCRIPTION
**Pull request by ttoine**

### CODE REVIEW

**Comment by ttoine** [comment]
I also created https://github.com/PrestaShop/core-weekly-generator/pull/27 to add ps_wirepayment to the list of known modules in the core weekly generator.

File: news/_posts/core-weekly/2020-03-23-coreweekly-week-12-2020.md [hunk 1]
```diff
@@ -0,0 +1,153 @@
...
+### Tests
+* [#18214](https://github.com/PrestaShop/PrestaShop/pull/18214): Upgrade TravisCI OS. Thank you [@mvorisek](https://github.com/mvorisek)
+* [#18077](https://github.com/PrestaShop/PrestaShop/pull/18077): Add behat test for GetManufacturerForViewingHandler. Thank you [@tdavidsonas88](https://github.com/tdavidsonas88)
+
+
+### Merge
+* [#18177](https://github.com/PrestaShop/PrestaShop/pull/18177): Merge 1.7.7.x into develop - 17/03/2020, by [@matks](https://github.com/matks)
+
+
+## Code changes in the '1.7.7.x' branch
```
Comment (matks) [inline news/_posts/core-weekly/2020-03-23-coreweekly-week-12-2020.md]:
```suggestion
## Code changes in the '1.7.7.x' branch (for v1.7.7.0)
```

**Review by matks — APPROVED** [review]
Provided one suggestion but it's not mandatory.
Dont forget to update publication date before merging 👍

File: news/_posts/core-weekly/2020-03-23-coreweekly-week-12-2020.md [hunk 2]
```diff
@@ -0,0 +1,153 @@
+---
+layout: post
+title:  "PrestaShop Core Weekly - Week 12 of 2020"
+subtitle: "An inside look at the PrestaShop codebase"
+date:   2020-03-23
```
Comment (ttoine) [inline news/_posts/core-weekly/2020-03-23-coreweekly-week-12-2020.md]:
```suggestion
date:   2020-03-24
```
Let's merge that today !

**Pull request merged**